packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -64,12 +65,15 @@ const SlotItem = ({
   const { t } = useLocale();
 
   const overlayCalendarToggled =
-    getQueryParam("overlayCalendar") === "true" || localStorage?.getItem("overlayCalendarSwitchDefault");
-  const [timeFormat, timezone] = useTimePreferences((state) => [state.timeFormat, state.timezone]);
+    getQueryParam("overlayCalendar") === "true" || localStorage.getItem("overlayCalendarSwitchDefault");
+  const [timeFormat, preferenceTimeZone] = useTimePreferences((state) => [state.timeFormat, state.timezone]);
   const bookingData = useBookerStore((state) => state.bookingData);
   const layout = useBookerStore((state) => state.layout);
+  const timezoneFromBookerStore = useBookerStore((state) => state.timeZone);
+  const { data: userSettings } = trpc.viewer.me.useQuery();
   const { data: eventData } = event;
   const hasTimeSlots = !!seatsPerTimeSlot;
+  const timezone = timezoneFromBookerStore ?? userSettings?.timeZone ?? preferenceTimeZone;
```

Comment:
user might not be login here to check for his preference. So. we need to remove /me query call as well from here as that would be 401 anyway for most booking scenarios